WordPress/Classes/ViewRelated/Post/PostSettings/Views/Excerpt/PostSettingsGenerateExcerptView.swift

```diff
@@ -0,0 +1,459 @@
+        case loading
+        case error
+        case finished
+    }
+
+    func testing(scenario: TestScenario) -> PostSettingsGenerateExcerptView {
+        var copy = self
+        copy.testScenario = scenario
+        return copy
+    }
+
+    private func setupTestScenario(_ scenario: TestScenario) {
+#if DEBUG
+        switch scenario {
+        case .loading:
+            isGenerating = true
+        case .error:
+            error = URLError(.unknown)
+        case .finished:
+            excerpts = [
+                GeneratedExcerpt(
+                    content: "Discover the cutting-edge trends transforming web development today. This comprehensive guide covers advanced JavaScript frameworks, innovative CSS techniques, and performance optimization strategies that modern developers use to create faster, more accessible websites.",
+                    isPartial: false
+                ),
+                GeneratedExcerpt(
+                    content: "Whether you're a seasoned developer or just starting out, this practical guide provides actionable insights into the latest web development trends. Learn about responsive design principles, modern frameworks, and techniques to build engaging digital experiences.",
+                    isPartial: false
+                ),
+                GeneratedExcerpt(
+                    content: "The future of web development is here! Explore innovative approaches to creating digital experiences, from advanced JavaScript frameworks to CSS techniques that enhance performance and accessibility. Get ready to transform your development workflow with these proven strategies.",
+                    isPartial: false
+                )
+            ]
+        }
+#endif
+    }
+}
+
+@available(iOS 26, *)
+@Generable
+private struct ExcerptGenerationResult {
+    @Guide(description: "Three different excerpt options, each capturing the main topic and key points of the post in a unique way")
+    var excerpts: [String]
+}
+
+private struct GeneratedExcerpt {
+    let content: String
+    var isPartial: Bool
+}
+
+@available(iOS 26, *)
+private struct ExcerptOptionView: View {
+    let index: Int
+    let excerpt: GeneratedExcerpt
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 8) {
+            HStack(alignment: .center) {
+                Text(Strings.optionLabel(index: index))
+                    .font(.subheadline.weight(.medium))
+
+                Spacer(minLength: 8)
+
+                if !excerpt.isPartial {
+                    HStack(alignment: .center, spacing: 4) {
+                        Text(Strings.characterCount(excerpt.content.count))
+                            .font(.footnote)
+                            .foregroundStyle(Color.secondary)
+
+                        Image(systemName: "chevron.right")
+                            .font(.caption2.weight(.semibold))
+                            .foregroundStyle(Color(.secondaryLabel).opacity(0.5))
```

Comment:
This chevron makes me think there is a details view upon tapping it. Can it be removed?